src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -501,14 +697,42 @@ def _deploy_arm_template_subscription_scope(cli_ctx,
     parameters = json.loads(json.dumps(parameters))
 
     properties = DeploymentProperties(template=template, template_link=template_link,
-                                      parameters=parameters, mode=mode)
+                                      parameters=parameters, mode=mode, on_error_deployment=on_error_deployment)
 
-    smc = get_mgmt_service_client(cli_ctx, ResourceType.MGMT_RESOURCE_RESOURCES)
-    if validate_only:
-        return sdk_no_wait(no_wait, smc.deployments.validate_at_subscription_scope,
-                           deployment_name, properties, deployment_location)
-    return sdk_no_wait(no_wait, smc.deployments.create_or_update_at_subscription_scope,
-                       deployment_name, properties, deployment_location)
+    return properties
+
+
+def _get_deployment_management_client(cli_ctx, handle_extended_json_format=False, aux_subscriptions=None):
+    smc = get_mgmt_service_client(cli_ctx, ResourceType.MGMT_RESOURCE_RESOURCES, aux_subscriptions)
+    deployments_operation_group = smc.deployments  # This solves the multi-api for you
```

Comment:
>deployments_operation_group [](start = 4, length = 27)

suggest to rename it as deployment_client, deployments_operation_group looks like a resource

Reply:
updated.